site/content/docs/standard/content-styles/hover-effects/index.html

```diff
@@ -50,6 +50,32 @@
   {{< /twsnippet/demo >}}
 
 
+  <!-- Section: Note about usage with animation  -->
```

Comment:
Please move the whole section before this ending tag in the docs:

```
</section>
<!-- Section: Basic example -->
```

It should be added after the code snippet.